### ISSUE TITLE
Viewlets should focus the first item, not the whole tree

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52000]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 2.1.1
Tools Used : Keyboard

**Repro Steps:**
1. Launch VS Code.
2. Navigate to Activity Bar and select "Explorer"(Cntrl+Shift+E) button.
3. Navigate to "File Explorer" treeview items using "Tab".

**Actual:**
Keyboard focus moves to overall treeview items which is non-interactive. Then using downward arrow keys focus goes to items in treeview.

**Expected:**
Keyboard focus should not move to the items as a whole which is non-interactive. The focus should move to first item in the treeview.

**Recommendations:**
Remove tab-index from the outer <div> that contains all the treeview items.
or,
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The keyboard only users will move to non-interactive elements on the screen which will be time consuming to reach only interactive elements on the page.

**MAS Reference:**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

**Attachment for Reference:**
[A11y_VSCode_ViewExplorer_Keyboard_TreeViewItems.pptx](https://github.com/Microsoft/vscode/files/2105535/A11y_VSCode_ViewExplorer_Keyboard_TreeViewItems.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
@JacksonKearl @sandy081 volunteres to tackle this?
Maybe we tackle this in one place (the view class which as a tree)?

**Comment by isidorn** [comment]
Actually this might be a tree issue, let's bring in @joaomoreno for opinion

**Comment by JacksonKearl** [comment]
I encountered this for the list view in fixing #53185 and address for list view in #53616. Tree view porbably shouldnt be too different. I can do it

**Comment by JacksonKearl** [comment]
I have a PR up that fixes this. But, as @bpasero mentioned in another thread, this removes keyboard access to the Explorer viewlet's Workspace Folder context menu.

**Comment by joaomoreno** [comment]
This is Viewlet behaviour, not tree/list. The fix should live in the Viewlet world.

**Comment by isidorn** [comment]
@JacksonKearl It seems like every viewlet has a different focus behavior, and we might have to fix this for each seperatly.
Code pointer for explorer viewlet as an example https://github.com/Microsoft/vscode/blob/e283c32b9db5faedfee01e30731edc6f21054bdf/src/vs/workbench/parts/files/electron-browser/views/explorerView.ts#L307

yyi next week I am on vacation, but I can also look into this once I am back in 10 days.

**Comment by JacksonKearl** [comment]
Same, but for lists: #53616

**Comment by JacksonKearl** [comment]
Closed with https://github.com/Microsoft/vscode/pull/53755#issuecomment-405499944.
